ui-test/test/specs/add-and-view-job-role.e2e.ts

```diff
@@ -0,0 +1,10 @@
+
+import ViewJobRolePage from '../pageobjects/view-job-roles.page.js'; // Import ViewJobRolePage class
+
+  describe('Job Role Viewing Process', () => {
```

Comment:
here indentation to the left (too many spaces or tabs) and whole describe block (lines 4-10)